css-shapes-2/Overview.bs

```diff
@@ -165,6 +168,79 @@ Supported Shapes</h3>
 					in the path string.
 					For the initial direction follow SVG 1.1.
 			</ul>
+		<dt>
+			<pre class=prod>
+				<dfn>shape()</dfn> = shape( [<<fill-rule>>]? from <<coordinate-pair>>, <<draw-command>>#)
+			</pre>
+		<dd dfn-type=value dfn-for="shape()">
+			<ul>
+				<li>
+					The <<coordinate-pair>> represents the starting point for the first draw-command,
+					in physical coordinates.
+				<li>
+					The sequence of <dfn><<draw-command>></dfn>s represent commands of an
+					<a href="https://www.w3.org/TR/SVG11/paths.html#PathData">SVG Path</a>.
+			</ul>
+	</dl>
+	
+	The arguments not defined above are defined as follows:
+
+	<dl>
+		<dt><dfn><<coordinate-pair>></dfn> = <<length-percentage>>{2}
+		<dd>Defines a pair of coordinates x & y.
+
+		<dt><dfn><<draw-command>></dfn> = <<move-command>> | <<line-command>> | <<hv-line-command>> |
+			<<curve-command>> | <<smooth-command>> | <<arc-command>> | close
+		<dd>
+			Defines a single draw command, equivalent to an <a href="https://www.w3.org/TR/SVG/paths.html#PathDataGeneralInformation">SVG draw command</a>.
+		<dt><dfn><<by-to>></dfn> = by | to
+		<dd>
+			Represents the reference from which offsets are computed in <<draw-command>>s.
+			When ''to'' is present, the coordinates are relative to the top-left origin of the reference box.
+			Otherwise ''by'' is present, the coordinates are relative to the end position of the previous command.
+
+			Note:
+			<<percentage>> values are always computed relative to the reference box regardless of how offsets are computed.
+		<dd>
+
+		<dt><dfn><<move-command>></dfn> = move <<by-to>> <<coordinate-pair>>
+		<dd>
+			Corresponds to a <a href="https://www.w3.org/TR/SVG/paths.html#PathDataMovetoCommands">moveto</a> command.
+		<dt><dfn><<line-command>></dfn> = line <<by-to>> <<coordinate-pair>>
+		<dd>
+			Corresponds to a <a href="https://www.w3.org/TR/SVG/paths.html#PathDataLinetoCommands">lineto</a> command.
+			If '''x''' or '''y''' are present instead of a <<length-percentage>>, the line will be horizontal or vertical, respectively.
+		<dt><dfn><<hv-line-command>></dfn> = [hline | vline] <<by-to>> <<length-percentage>>
+		<dd>
+			Corresponds to a horizontal or vertical <a href="https://www.w3.org/TR/SVG/paths.html#PathDataLinetoCommands">lineto</a> command.
+			<dt><dfn><<curve-command>></dfn> = curve <<by-to>> <<coordinate-pair>> via <<coordinate-pair>>{1,2}
```

Comment:
would prefer `using` over `via`. The curve does not go via the control points.

Reply:
Also, I wonder if the end point should come after the control points.

Reply:
Lastly (hopefully), additional spec text should point out that an animation of the `shape` would work the same as SVG, but that `curve` is a bit different since the type of curve is defined by the number of parameters (rather than an identifier like SVG).